src/components/Profile/components/OfficeInfoList/index.jsx

```diff
@@ -110,24 +104,9 @@ const OfficeInfoList = ({
       }
     />
   ) : Mail_Location ? (
-    <ProfileInfoListItem title="Mailstop:" contentText={{ Mail_Location }} />
+    <ProfileInfoListItem title="Mailstop:" contentText={Mail_Location} />
   ) : null;
 
-  const updateOfficeInfo =
-    myProf && PersonType?.includes('fac') ? (
-      <Typography align="left" className={styles.note}>
-        NOTE: Update your office info{' '}
-        <a
-          href="https://go.gordon.edu/general/myaccount.cfm"
-          target="_blank"
-          rel="noopener noreferrer"
-          style={{ color: gordonColors.primary.blue }}
-        >
-          here
-        </a>
-      </Typography>
-    ) : null;
-
```

Comment:
Should we be removing the update?

Reply:
(nevermind, this was resolved by @antoniavonto )

Reply:
Antonia confirmed that this is old and unused code.